runit/Dockerfile

```diff
@@ -18,8 +18,8 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get -y update && \
     ruby2.0 \
     zip && \
   DEBIAN_FRONTEND=noninteractive apt-get -y install --no-install-recommends collectd-core && \
-  apt-get clean && \
-  rm -rf /var/lib/apt/lists/*
+  DEBIAN_FRONTEND=noninteractive apt-get clean && \
```

Comment:
I think the trailing `\` on line 20 carries over to here, so you shouldn't need to repeat the `DEBIAN_FRONTEND=noninteractive` part.

Reply:
updated.